ansible/roles/gen_kubernetes_config/templates/testnet-homepage.yaml.j2

```diff
@@ -0,0 +1,148 @@
+# ANSIBLE GENERATED FILE - DO NOT EDIT MANUALLY
+testnet-homepage:
+  fullnameOverride: homepage
+
+  replicas: 1
+
+  image:
+    pullPolicy: Always
+    repository: ethpandaops/ethereum-testnet-homepage
+
+  resources:
+    requests:
+      cpu: 33m
+      memory: 64Mi
+  ingress:
+    enabled: true
+
+    className: ingress-nginx-public
+
+    hosts:
+      - host: {{ ethereum_network_name }}.ethpandaops.io
+        paths:
+          - path: /
+            pathType: Prefix
+
+  customMarkdown: |-
+    ## Network Spec
+    https://notes.ethereum.org/@parithosh/{{ ethereum_network_name  | replace('4844-', '')}}-specs
```

Comment:
Created a combined team on notes.ethereum.org called ethpandaops. We can now host stuff as https://notes.ethereum.org/@ethpandaops/test for e.g.. So i'd hardcode the team name and all future specs can end up there. I've invited everyone as admin, you can move over whatever notes you like into the ethpandaops namespace.

Reply:
Could you add devnet 7 already. so I can point to it?